### ISSUE TITLE
Limit create account

### ISSUE THREAD

**Issue by antinsp** [Leiren/Npanel#31]
how many accounts can I make on npanel, in the past I tried 30 accounts, is it still the same now or has it been added, it's a shame that I can only have 30 accounts, I admit at least 60 new accounts are good

**Comment by isp85** [comment]
I agree with the author here. How can we increase the limit from 30?

FYA @Leiren
